<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>catalogbook</class>
 <widget class="QMainWindow" name="catalogbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>АРМ Каталог книг</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leSearchQuery">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>17</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Ваш поисковый запрос (допускается поиск по названию, по серии, по автору)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>21</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поиск:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="twCatalogList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>1011</width>
      <height>511</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_Search">
     <attribute name="title">
      <string>Поиск по каталогу</string>
     </attribute>
     <widget class="QTableWidget" name="twBookCatalog">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>961</width>
        <height>481</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView {
    background-color: #fff;
    font-size:13px;
}
QHeaderView::section:horizontal {
    color: #fff;
    border-style: solid;
    background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #4287ff, stop: 1 #356ccc);
 }
QTableView {
    border: 2px solid #3873d9;
    border-top-color: #4287ff;
    border-radius: 4px;
    background-color: #fff;
    gridline-color: #777;
    selection-background-color: #ccdfff;
    color:#333;
    font-size:12px;
 }

QScrollBar:vertical {
     background: #e4e4e4;
     border-top-right-radius: 4px;
     border-bottom-right-radius: 4px;
     width: 12px;
     margin: 0px;
 }
 QScrollBar::handle:vertical {
    background-color: qlineargradient( x1: 0, y1: 0, x2: 1, y2: 0,
                                    stop: 0 #4287ff, stop: 1 #356ccc);
     border-radius: 4px;
     min-height: 20px;
     margin: 0px 2px 0px 2px;
 }
 QScrollBar::add-line:vertical {
     background: none;
     height: 0px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
     background: none;
     height: 0px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }</string>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Автор</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Заглавие</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Серия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Заказать?</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Locate</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_FullDescription">
     <attribute name="title">
      <string>Описание книги</string>
     </attribute>
     <widget class="QGroupBox" name="gbCOver">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lblCover">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>271</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>701</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Аннотация</string>
      </property>
      <widget class="QTextEdit" name="teAnnotation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>681</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>951</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Библиографические сведения</string>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>331</width>
         <height>321</height>
        </rect>
       </property>
       <property name="title">
        <string>Содержание</string>
       </property>
       <widget class="QTreeView" name="tvContain">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>30</y>
         <width>301</width>
         <height>321</height>
        </rect>
       </property>
       <property name="title">
        <string>Место хранения</string>
       </property>
       <widget class="QListWidget" name="lwLocate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>281</width>
          <height>281</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="leDelay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="vsDelay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>23</width>
      <height>271</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLimit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>660</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDelay">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>330</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Задержка</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLimit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>52</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMIT</string>
    </property>
   </widget>
   <widget class="QSlider" name="hsTimerBoost">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>861</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBoost">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>70</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFilterAddress">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>17</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Каталог</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Обновить вручную</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
